### ISSUE TITLE
Multiple EXTRA_PACKAGES

### ISSUE THREAD

**Issue by madwort** [mhausenblas/mkdocs-deploy-gh-pages#73]
I'm attempting to use this for a repo that requires at least three extra apk packages - `build-base`, `openblas-dev`, and `lapack-dev`, but when I set:
```
EXTRA_PACKAGES: "build-base openblas-dev lapack-dev"
```
I get:
```
ERROR: 'build-base openblas-dev lapack-dev' is not a valid world dependency, format is name(@tag)([<>~=]version)
``` 
I think this is because of the quotes:

https://github.com/mhausenblas/mkdocs-deploy-gh-pages/blob/master/action.sh#L9-L11

Is there a reason this isn't just `apk add --no-cache ${EXTRA_PACKAGES}`?

**Comment by madwort** [comment]
For what it's worth, it turned out to be much better for our use case to switch to https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-static-site-generators-with-python instead - although this action worked, it was taking 1hr to compile all the dependencies (numpy, pandas, etc) - using an ubuntu base image (& a pip cache) means that our dependencies can be downloaded as wheels & install in seconds.

**Comment by mhausenblas** [comment]
Thanks for the feedback and glad you found something that works best for you, @madwort! Appreciate all your contribs here.

**Comment by atodorov** [comment]
@mhausenblas do you mind tagging a new version so we can make use of the fix ?

**Comment by mhausenblas** [comment]
@atodorov thanks for the reminder! Done in 1.17

**Comment by Trymzet** [comment]
Hi, this is still broken on master:
`ERROR: 'gcc libffi-dev' is not a valid world dependency, format is name(@tag)([<>~=]version)`

**Comment by madwort** [comment]
@Trymzet this was fixed, but unfortunately the fix has since been reverted

**Comment by mhausenblas** [comment]
Should be fixed in 1.21 thanks to @madwort
